src/hooks/useFiatText.ts

```diff
@@ -70,7 +70,7 @@ export const formatFiatString = (props: { autoPrecision?: boolean; fiatAmount: s
     }
     while (tempFiatAmount <= 0.1 && tempFiatAmount > 0) {
       tempFiatAmount *= 10
-      precision++
+      precision += precision < maxPrecision ? 1 : 0
```

Comment:
I don't quite know what's going on here. But I'll trust that you've got it to work. I did make a fixup commit to fix the types that appear to be incorrect.

Reply:
This dynamically increases precision as the fiat value gets smaller. Not really applicable to any of our current fiat cases, so I just changed it to do the same thing, but flip it around to cap at a maxPrecision for future-proofing